src/schema/v2/me/mark_all_notifications_as_read_mutation.ts

```diff
@@ -7,11 +7,19 @@ import {
   GravityMutationErrorType,
 } from "lib/gravityErrorHandler"
 import { ResolverContext } from "types/graphql"
+import { meType } from "../me"
 
 const SuccessType = new GraphQLObjectType<any, ResolverContext>({
   name: "MarkAllNotificationsAsReadSuccess",
   isTypeOf: (data) => data.success,
   fields: () => ({
+    me: {
+      type: new GraphQLNonNull(meType),
+      resolve: (_source, _args, { meLoader }) => {
+        debugger
```

Comment:
👀

Reply:
I have created a PR to add an ESLint rule to avoid committing `debugger`: https://github.com/artsy/metaphysics/pull/5574